Makefile

```
@@ -90,6 +90,14 @@ generate-in-docker:
 format: $(GOIMPORTS) $(GOIMPORTSREVISER)
 	@bash $(GARDENER_HACK_DIR)/format.sh ./cmd ./pkg ./test
 
+.PHONY: sast
+sast: tidy $(GOSEC)
```

We don't use tidy to copy the script in this repo, hence the `sast` target should not require `tidy`, right?

```suggestion
sast: $(GOSEC)
```